obsfit/cost_obsfit.F

```diff
@@ -162,10 +162,10 @@ SUBROUTINE COST_OBSFIT(
 C Combine all processes
          DO ii=1,NSAMPLESMAX
           tmpgs = samples_buff(ii)
-          _GLOBAL_SUM_RL(tmpgs, myThid) 
+          _GLOBAL_SUM_RL (tmpgs, myThid) 
```

Comment:
TIP: Adding a space may cause a compilation error on Darwin 22+ machines. For some reason the macro functions don't work the same in MacOS versus Linux.


@shoshanareich

Reply:
Thanks for catching this!